sources/packages/web/src/views/institution/AppInstitution.vue

```diff
@@ -38,11 +37,13 @@ export default {
         });
       } else {
         if (await UserService.shared.checkUser()) {
-          await InstitutionService.shared.sync();
-          if (route.path === AppRoutes.InstitutionRoot) {
-            router.push({
-              name: InstitutionRoutesConst.INSTITUTION_DASHBOARD,
-            });
+          if (await UserService.shared.checkActiveUser()) {
```

Comment:
Do we need to check if the user is active at this point?
I believe that if it is not the user will be logged of before reaching this point, doesn't it?
f not, keep it, and please consider this comment.

Reply:
INSTITUTION_DASHBOARD should only be shown to active users,  right? that's why I add that check there